ui/src/components/Navbar/Navbar.scss

```diff
@@ -31,7 +31,7 @@
     }
 
     .heading-title {
-      padding-right: 18.5em;
+      width: 21.5em;
```

Comment:
Is this regarding the line break in the current name? Is there a solution that doesn't hard-code a width at all?